### ISSUE TITLE
Migration from v1 to v2

### ISSUE THREAD

**Issue by gsouf** [nshenderov/strapi-plugin-ckeditor#107]
Hi, I'm reading the doc for the new v2 in the readme, I can't understand the new configuration.

It is mentioned

```
Each config includes: 
    "styles", "field" and "editorConfig" properties.
    Property "field" is required.
```

What are each of these properties for? Are they specific to ckeditor,  or to strapi, or is it custom config of the plugin?

For example in the given config file I see a field `configs.toolbar.editorConfig` and also a field `custom.editorConfig.toolbar`. I'm not sure what is used where?

Thanks

**Comment by gsouf** [comment]
@nshenderov "not planed"?

**Comment by nshenderov** [comment]
@gsouf The issue is stale. Is it still relevant to you?

**Comment by gsouf** [comment]
@nshenderov I solved by experimenting but I can't remember what I did as it's working and I haven't touched it for months. Give 8 people gave a thumb up, I suppose other people are unable to understand the documentation. It's just a matter of clarifying what is written in the README

**Comment by nshenderov** [comment]
@gsouf I see.
So the big idea here is that you can have multiple configurations of the editor and choose the one you need currently in the content type builder. The plugin provides four configurations by default, which you can modify, and you can also add new configurations.
The plugin has many configuration options, as does CKEditor inside it, and I completely agree that the documentation needs further improvement and that the naming might be better. It takes time to understand the documentation and how to configure the plugin properly, but most people did understand how it works.
If you have some time to help improve it, that would be great.

**Comment by gsouf** [comment]
@nshenderov thanks for the explanation, today I checked our codebase to see what we did. I just remebered that we decided to uninstall this plugin and use the official strapi CKEditor plugin instead as it was faster to set up for us.

The major porblem is that I really had no idea what the given config were for. I see that you added some docs to the README.md, that's probably going to help for the future
